.eslintrc

```diff
@@ -1,12 +1,13 @@
 # https://github.com/eslint/eslint/blob/master/docs/rules/README.md
 
 parserOptions:
-  ecmaVersion: 2015
+  ecmaVersion: 2017
```

Comment:
ES 2017 isn't supported in older versions of browsers where Stylus can run, AFAIK.

Reply:
The build scripts are only run by us and they are written using async; I'm not sure how to differentiate the extension code ecma version & the build code version.

Reply:
Well, you'll have to exclude the build script from Stylus eslint config or rewrite without async.

Reply:
Ok, fixed... I'm not sure why but now I'm only seeing the one `no-unused-vars` message.

Reply:
Maybe because of `env: node`

Reply:
yeah, maybe!